.gitattributes

```diff
@@ -0,0 +1,7 @@
+/tests export-ignore
```

Comment:
I've double checked with one of the drivers (see https://github.com/minkphp/MinkSelenium2Driver/blob/master/phpunit.xml.dist#L7) and it seems, that only `driver-testsuite` folder is included in PHPUnit. So this should work theoretically.